homework/week-0/AllisonBojarski/AnimalSort(HWproblem#1)/src/nyc/c4q/alizinha/DomesticCat.java

```diff
@@ -0,0 +1,37 @@
+package nyc.c4q.alizinha;
+/**
+ * Created by c4q-Allison on 5/1/15.
+ */
+public class DomesticCat extends Animal implements Domestic
+{
+
+    String name;
+
```

Comment:
Just an echo of the above comment about a constructor or two. In this case, you have your setter for the purpose of this class according to the assignment instructions.